tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,668 @@
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test Filters Package System tests:
+        foglamp-south-http-south south plugin
+        foglamp-filter-expression, foglamp-filter-python3, foglamp-filter-simple-python, foglamp-filter-ema filter plugins
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import subprocess
+import http.client
+import json
+import pytest
+import os
+import time
+import urllib.parse
+
+PROJECT_ROOT = os.environ.get("PROJECT_ROOT")
+SCRIPTS_DIR_ROOT = "{}/tests/system/python/scripts/package".format(PROJECT_ROOT)
+ASSET_NAME_PY35 = "end_to_end_py35"
+ASSET_NAME_SP = "end_to_end_sp"
+ASSET_NAME_EMA = "end_to_end_ema"
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 20
+
+
+def post_request(foglamp_url, post_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("POST", post_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    res = res.read().decode()
+    r = json.loads(res)
+    return r
+
+
+def get_request(foglamp_url, get_url):
+    con = http.client.HTTPConnection(foglamp_url)
+    con.request("GET", get_url)
+    resp = con.getresponse()
+    r = json.loads(resp.read().decode())
+    return r
+
+
+def put_request(foglamp_url, put_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("PUT", put_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    r = json.loads(res.read().decode())
+    return r
+
+
+# TODO : set PROJECT_ROOT appropriately
+def call_fogbench(wait_time):
+    execute_fogbench = 'cd {}/extras/python ;python3 -m fogbench -t $FOGLAMP_ROOT/data/template.json ' \
+                       '-p http -O 10'.format(PROJECT_ROOT)
+    exit_code = os.system(execute_fogbench)
+    assert 0 == exit_code
+    time.sleep(wait_time)
+
+
+def add_south_http(foglamp_url, name):
+    data = {"name": name, "type": "south", "plugin": "http_south", "enabled": True, "config": {}}
```

Comment:
config is an optional parameter. Here we are adding south service with default config. Therefore No need of this KV